client/my-sites/importer/error-pane.jsx

```diff
@@ -76,17 +77,27 @@ class ImporterError extends PureComponent {
 	};
 
 	getPreUploadError = () => {
-		return this.props.translate(
-			'You have uploaded a .wpress file that works with the All-in-One WP Migration plugin. You can either {{ip}}install that plugin{{/ip}}, or {{ei}}try out Everything Import{{/ei}}. {{cs}}Still need help{{/cs}}?',
-			{
-				components: {
-					br: <br />,
-					ip: <Button className="importer__error-pane is-link" onClick={ this.installPlugin } />,
-					ei: <Button className="importer__error-pane is-link" onClick={ this.everythingImport } />,
-					cs: <Button className="importer__error-pane is-link" onClick={ this.contactSupport } />,
-				},
-			}
+		const defaultError = this.props.translate(
+			'Oops! We ran into an unexpected error while uploading your file.'
 		);
+
+		if ( this.props.code === 'WPRESS_FILE_IS_NOT_SUPPORTED' ) {
+			return this.props.translate(
+				'You have uploaded a .wpress file that works with the All-in-One WP Migration plugin. You can either {{ip}}install that plugin{{/ip}}, or {{ei}}try out Everything Import{{/ei}}. {{cs}}Still need help{{/cs}}?',
+				{
+					components: {
+						br: <br />,
```

Comment:
Do we need this br here? It looks like there's none in the sentence above?

Reply:
Good catch! The useless code was removed at https://github.com/Automattic/wp-calypso/pull/66674/commits/07674dba9776c28b4c5fb3fa4d69e57f631e4f06